docs/guides/theme_architecture/partials.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Partials
+parent: Theme architecture
+nav_order: 7
+---
+
+# Partials
+
+Partials allow you to extract commonly used code snippets so you only have to define them once. You can reference these snippets throughout the theme with the Liquid render tag.
+
+##### syntax
+{% raw %}
+```liquid
+{% render 'partial' %}
+```
+{% endraw %}
+
+Where `partial` is the name of the snippet to render, without the `.html` extension.
+
+Inside snippets you can't directly access variables that are created outside of the snippet. However, you can specify variables as parameters to pass outside variables into snippets. Any changes that are made to a passed variable apply only within the snippet.
+
+##### syntax
+{% raw %}
+```liquid
+{% render 'partial', variable: value %}
```

Comment:
I'd pass at least 2 variables in for the example, so it's clear how to separate them

Reply:
Good thinking